css/css-syntax-3/tokenizing-and-parsing/input-byte-stream/i18n-css-syntax-bytes-001.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="ISO-8859-15"/>
```

Comment:
Do we need both a meta element and a header?

Reply:
yes